### PR TITLE
release-checklist: update packaging section

### PR DESCRIPTION
**Pull request by arithx**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -9,7 +9,19 @@ Tagging:
...
+   - Bump the `Version`
+   - Switch the `Release` back to `1.git%{shortcommit}%{?dist}`
+   - Remove any patches obsoleted by the new release
+   - Run `go-mods-to-bundled-provides.py | sort` while inside of the Ignition directory you ran `./tag_release` from & copy output into spec file in `# Main package provides` section
+   - Add any new spec paths to `%gotest` lines
+   - Update change log
+ - [ ] Run `./download_ignition.sh`
+ - [ ] Run `kinit your_fas_account@FEDORAPROJECT.ORG`
+ - [ ] Run `fedpkg upload tarball-name`
+ - [ ] Edit `./sources` to delete the old tarball
```
Comment (jlebon) [inline .github/ISSUE_TEMPLATE/release-checklist.md:23]: You can combine those two with `fedpkg new-sources tarball-name`.

**Review by jlebon — APPROVED** [review]
One comment, LGTM otherwise!

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 1 above)
Comment (arithx) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: nifty, updated.

**Review by bgilbert — APPROVED** [review]

**Review by sohankunkerkar — APPROVED** [review]
@arithx Thanks for the write-up

**Pull request merged**